src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivity.test.js

```diff
@@ -127,30 +127,18 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity',
         expect(wrapper.exists('CommentForm')).toBe(false);
     });
 
-    test('should correctly render annotation activity of another file version', () => {
-        const wrapper = getWrapper({ isCurrentVersion: false });
-
-        expect(wrapper.find('AnnotationActivityLink').prop('message')).toEqual({
-            ...messages.annotationActivityVersionLink,
-            values: { number: '2' },
-        });
-    });
-
-    test('should render version unavailable if file version is null', () => {
-        const wrapper = getWrapper({ item: { ...mockAnnotation, file_version: null } });
-        const activityLink = wrapper.find('AnnotationActivityLink');
-
-        expect(activityLink.prop('message')).toEqual({
-            ...messages.annotationActivityVersionUnavailable,
-        });
-        expect(activityLink.prop('isDisabled')).toBe(true);
-    });
-
     test('should render commenter as a link', () => {
         const wrapper = getWrapper();
         expect(wrapper.find('UserLink').prop('name')).toEqual(mockActivity.item.created_by.name);
     });
 
+    test('should hide AnnotationActivityLink if hasVersions is false', () => {
```

Comment:
Should we also add a test case for when `hasVersions` is true? Or would the `AnnotationActivityLink` still be hidden in that case?